package.json

```diff
@@ -398,11 +395,11 @@
     "@storybook/addon-essentials": "^6.3.0",
     "@storybook/addon-links": "^6.3.0",
     "@storybook/react": "^6.3.0",
-    "babel-loader": "^8.1.0",
+    "babel-loader": "8.2.2",
     "full-icu": "1.4.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "27.4.5",
-    "jest-environment-jsdom-sixteen": "1.0.3",
+    "jest": "26.6.3",
```

Comment:
I did a final check, I think Tiberiu bumped this to 27 at some point for some reason. 
Is there any limitation that we should stick to 26?

Reply:
ok, razzle breaks :) It seems that something is wrong in the API after 26, and the razzle test script:

```
TypeError: Cannot read properties of undefined (reading 'onlyChanged')
    at _default (/Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/getChangedFilesPromise.js:49:20)
    at /Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:296:71
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:108:24)
    at _next (/Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:128:9)
    at /Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:133:7
    at new Promise (<anonymous>)
    at /Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:125:12
    at _run (/Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:374:20)
    at /Users/sneridagh/Development/plone/volto/node_modules/razzle/node_modules/@jest/core/build/cli/index.js:178:13
```

@tiberiuichim what do you think? is it fine for you to get back to 26?

Reply:
I was just trying to match what Razzle is using. Yeah, let's see what @tiberiuichim thinks...

Reply:
Tiberiu said it was ok for now. We can push for having it working later if needed. I added the breaking notice as well.